mass_mailing_disable_tracking/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# SPDX-FileCopyrightText: 2025 hugues de keyzer
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
I didn't know about SPDX before ! I suppose it's fine to use it in OCA until CI is green :)

Reply:
I know @sbidoul had exchanges with @carmenbianca some years ago about that and to include them in our guidelines. But I don't know the resulting conclusions.

Reply:
I remember Carmen mentioning that at some OCA days, but I don't think there was any conclusion and I have not heard about it since. These comments do not harm I guess, but I honestrly don't know what it would take to make that useful on OCA scale.